### PR TITLE
Create zookeeper node if it doesn't exist

### PR DESCRIPTION
**Pull request by tsegaran**
:eyeglasses: @rram

### CODE REVIEW

File: reddit_adzerk/adzerkkeywords.py [hunk 1]
```diff
@@ -20,4 +22,6 @@ def update_global_keywords():
 
     # Store results in zookeeper
     if g.zookeeper:
-        g.zookeeper.set("/keyword-targets", json.dumps(list(keyword_target)))
+        if not g.zookeeper.exists(KEYWORD_NODE):
```
Comment (rram) [inline reddit_adzerk/adzerkkeywords.py]: Sorry, I should have given you more direction. We usually use `g.zookeeper.ensure_path()`

**Comment by rram** [comment]
:nail_care:

**Comment by tsegaran** [comment]
Ok, using ensure_path now. :haircut:

**Comment by rram** [comment]
:fish:

**Comment by tsegaran** [comment]
675c744

**Pull request closed without merging**